samples/notebooks/python/6-memory-and-embeddings.ipynb

```diff
@@ -394,9 +394,9 @@
     "i = 0\n",
     "for memory in memories:\n",
     "    print(\"Result {++i}:\")\n",
-    "    print(\"  URL:     : \" + memory.Id)\n",
-    "    print(\"  Title    : \" + memory.Description)\n",
-    "    print(\"  Relevance: \" + memory.Relevance)\n",
+    "    print(\"  URL:     : \" + memory.id)\n",
+    "    print(\"  Title    : \" + memory.description)\n",
+    "    print(\"  Relevance: \" + memory.relevance)\n",
```

Comment:
Just did a quick test here, looks like it needs to be `str(memory.relevance)` (to run the notebook w/out issues, otherwise, testing as is, I get an error when it tries to concat a `numpy.foat64` to a `str`).
```suggestion
    "    print(\"  Relevance: \" + str(memory.relevance))\n",
```

Reply:
@jjhenkel I hope the commit I submitted matches what you intended :)